lib/bank-player.js

```diff
@@ -38,10 +43,10 @@ function playBuffer (ctx, destination, buffer, time, duration, gain) {
   var vca = ctx.createGain()
   source.connect(vca)
   vca.connect(destination)
-  vca.gain.linearRampToValueAtTime(gain, time)
+  vca.gain.linearRampToValueAtTime(gain, ctx.currentTime)
   if (duration > 0) {
     source.start(time, 0, duration)
-    vca.gain.exponentialRampToValueAtTime(0.1, time + duration)
+    vca.gain.exponentialRampToValueAtTime(0.01, gain, ctx.currentTime + time + duration)
```

Comment:
Yes, its not an offset, it's absolute time (I think this is the way it works on most Web API functions)...